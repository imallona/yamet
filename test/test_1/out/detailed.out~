chr     start   end     simulations.tsv
chrM    1       201     0
chrU    1       201     -1
chr1_1  1       201     -1
chr3_1  1       201     -1
chr_random      1       201     -1
