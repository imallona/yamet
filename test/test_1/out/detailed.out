chr	start	end	test_1/in/simulations.tsv	shannon	avg_meth
chrM	1	201	0	0	1
chrU	1	201	0	0	0
chr1_1	1	201	0	0.499915	0.340206
chr3_1	1	201	0.437214	0.746068	0.604938
chr_random	1	201	0.742137	0.995618	0.340206
